### PR TITLE
chore: Update master from release v0.30

### PR DESCRIPTION
**Pull request by Ivansete-status**
### Description
This PR is aimed to put the `release/v0.30` changes into `master` so that we keep them for future versions

### CODE REVIEW

File: CHANGELOG.md
Comment (rymnc) [inline CHANGELOG.md:12]:
```suggestion
`N` messages per epoch, instead of just one message per epoch. See [this](https://github.com/waku-org/nwaku/issues/2345) for more details.
```

File: CHANGELOG.md
Comment (rymnc) [inline CHANGELOG.md:17]:
```suggestion
- rln: fix nullifierlog vulnerability ([#2855](https://github.com/waku-org/nwaku/pull/2855))
```

**Review by rymnc — APPROVED** [review]
LGTM if checks pass

**Review by gabrielmer — APPROVED** [review]
LGTM, thanks so much! 🤩

**Comment by Ivansete-status** [comment]
Sorry for the mess guys. After merging that, I realized I didn't use the correct merge button.
Then I reverted the change by force-pushing master to the correct commit: https://github.com/waku-org/nwaku/commit/cae0c7e378be688dbb464f2f19ac6aba1c59b858

And I will repeat the same PR but using the correct option so that we get the correct version after calling `git describe --tags`.

**Pull request merged**
